src/scripts/medgen_conflicts_add_xrefs.py

```diff
@@ -0,0 +1,100 @@
+"""MedGen conflicts: Add new xrefs
```

Comment:
Hey @nicolevasilevsky. The PR requires you to review because you are a code owner. I think Nico has this one covered but if you have time and want to glance, feel free!

From `.github/CODEOWNERS`:

```
* @nicolevasilevsky
*.sparql @matentzn
*.py      @joeflack4
```

@matentzn @nicolevasilevsky To me, this looks like Nicole will be required on every single PR. Is this what we want?

Reply:
## `mondo-edit.obo` conflict
@matentzn When you get around to reviewing this, there may be a conflict with `mondo-edit.obo`. I had been rebasing this branch, resetting that file, and then re-running `sh run.sh make address-medgen-conflicts-aug2023-deletes`. When you get to reviewing this, let me know and I can do that again for you, or you can do what I just described or some variation of it.

Reply:
Thank you, I will take it from here!